.eslintrc

```diff
@@ -1,12 +1,7 @@
 {
-  "extends": ["standard"],
+  "extends": ["semistandard"],
```

Comment:
Yeah, agree. What's the best way to avoid that? I think most of the times, these things are tied to tickets, but this initial change was just a PR.